### ISSUE TITLE
tests fail with lalsuite 6.71: "not a code object"

### ISSUE THREAD

**Issue by dbkeitel** [PyFstat/PyFstat#52]
Since today, even after reverting master to last known working state, with
```
 INTERNALERROR> TypeError: not a code object: <class 'pyfstat.make_sfts.Writer'>
```
I suspect some changes in the github python versions or package dependencies correlated with the design changes, and that somehow makes the syntax checker more strict...?

**Comment by dbkeitel** [comment]
Comparing the last working pipeline outputs with today's, at first I though it was that the test time dependencies had been updated, but [this](https://github.com/PyFstat/PyFstat/pull/55/files#diff-7327122c763912b429300af1b6158a12) didn't help. Instead, it's the lalsuite 6.71 relese that somehow messed things upm and #56 has fixed things for now.

Need to dig into this...

**Comment by dbkeitel** [comment]
Actually, doing a `pip install --upgrade lalsuite` in my local venv, I can now reproduce this! That will make things easier to debug.

**Comment by dbkeitel** [comment]
Fixed by lalsuite-6.72, see #59.
